src/student_success_tool/modeling/utils.py

```diff
@@ -94,11 +95,17 @@ def load_features_table(rel_fpath: str) -> dict[str, dict[str, str]]:
         rel_fpath: Path to features table TOML file relative to package root;
             for example: "assets/pdp/features_table.toml".
     """
-    pkg_root_dir = next(
-        p
-        for p in pathlib.Path(__file__).parents
-        if p.parts[-1] == "student_success_tool"
-    )
+    try:
+        # If __file__ is defined, we're in a regular script or module
+        pkg_root_dir = next(
+            p
+            for p in pathlib.Path(__file__).parents
+            if p.parts[-1] == "student_success_tool"
+        )
+    except NameError:
+        # If __file__ is not available (i.e. Jupyter notebook), use the current working directory
```

Comment:
I'm very surprised by this! I thought `__file__` refers to the path to this module, no matter the env in which it's being imported. In fact I don't believe I've ever had this issue in a Notebook.